eventually/tests/unittests/validators/testvalidators.py

```diff
@@ -124,6 +133,627 @@ def test_password_validator_fail(self):
+        """
+        data = {
+            'title': "some_title",
+            'description': "some_description",
+            'status': "some_status",
+            'add_users': [1, 2, 3],
+            'remove_users': [4, 5, 6]
+        }
+
+        is_valid = task_data_validate_update(data)
+
+        self.assertFalse(is_valid)
+
+    def test_task_data_validate_update_fail_add_users_invalid_type(self):
+        """
+        Method that tests `task_data_validate_update`,
+        when `data['add_users']` is neither list nor tuple.
+        """
+        data = {
+            'title': "some_title",
+            'description': "some_description",
+            'status': 1,
+            'add_users': "[1, 2, 3]",
+            'remove_users': [4, 5, 6]
+        }
+
+        is_valid = task_data_validate_update(data)
+
+        self.assertFalse(is_valid)
+
+    def test_task_data_validate_update_fail_remove_users_invalid_type(self):
+        """
+        Method that tests `task_data_validate_update`,
+        when `data['remove_users']` is neither list nor tuple.
+        """
+        data = {
+            'title': "some_title",
+            'description': "some_description",
+            'status': 1,
+            'add_users': [1, 2, 3],
+            'remove_users': "[4, 5, 6]"
+        }
+
+        is_valid = task_data_validate_update(data)
+
+        self.assertFalse(is_valid)
+
+    def test_profile_data_validator_success(self):
+        """
+        Method that tests `profile_data_validator`.
+        """
+        data = {
+            "hobby": "swimming",
+            "birthday": "180110",
+            "photo": "png"
+        }
+
+        is_valid = profile_data_validator(data)
+
+        self.assertTrue(is_valid)
+
+    def test_profile_data_validator_fail_hobby_invalid_type(self):
+        """
+        Method that tests `profile_data_validator`,
+        when `data['hobby']` is not a string.
+        """
+        data = {
+            "hobby": 222
+        }
+
+        is_valid = profile_data_validator(data)
+
+        self.assertIsNone(is_valid)
+
+    def test_profile_data_validator_fail_birthday_invalid_type(self):
+        """
+        Method that tests `profile_data_validator`,
+        when `data['birthday']` is not a string.
+        """
+        data = {
+            "birthday": 222
+        }
+
+        is_valid = profile_data_validator(data)
+
+        self.assertIsNone(is_valid)
+
+    def test_profile_data_validator_fail_photo_invalid_type(self):
+        """
+        Method that tests `profile_data_validator`,
+        when `data['photo']` is not a string.
+        """
+        data = {
+            "photo": 222
+        }
+
+        is_valid = profile_data_validator(data)
+
+        self.assertIsNone(is_valid)
+
+    def test_valid_date_type_success_valid_format_Ymd(self):
+        """
+        Method that tests `valid_date_type`,
+        when `datec` format matches '%Y%m%d'.
+        """
+        datec = "20180110"
```

Comment:
why not date?

Reply:
Because parameter in method valid_date_type (eventually/utils/validators.py file) has such name ('datec'). Should I change it to 'date' only in tests or in method as well?

Reply:
`date` is more meaningful, but I remember that there was some reason why we chose `datec` name. If it is possible try change this name in validation and test files, but if not leave it as it is.

Reply:
Done!
Modified 'datec' to 'date' in validation and in test files.